### PR TITLE
Added docker files to enable containerization of wrapper and database.

### PR DESCRIPTION
**Pull request by atulai-sg**
- Added docker files to enable containerization of wrapper and database.
- Added instructions to start wrapper and mongodb containers.

The changes are superset of https://github.com/NHA-ABDM/ABDM-wrapper/pull/33 and https://github.com/NHA-ABDM/ABDM-wrapper/pull/34.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM eclipse-temurin:17-jdk-alpine
+VOLUME /tmp
+COPY build/libs/*-SNAPSHOT.jar app.jar
```
Comment (chandrashekar-s) [inline Dockerfile:3]: Building the files and then copying to the container will make sure we pick the latest changes.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+  abdm-wrapper:
+    build: .
+    image: abdm-wrapper
+    ports:
+      - '8080:8080'
```
Comment (chandrashekar-s) [inline docker-compose.yaml:8]: Generally I have noticed people exposing a different port rather than the default port 8080 to avoid conflicts.

**Review by chandrashekar-s — COMMENTED** [review]
@atulai-sg Thanks for the changes, there are 2 minor comments/suggestions. Please have a look.

File: Dockerfile (hunk 1 above)
Comment (atulai-sg) [reply Dockerfile]: I agree, made the changes.

**Review by chandrashekar-s — APPROVED** [review]
@atulai-sg Thanks for addressing the comments. The changes look good.

**Pull request merged**